src/ci/github-actions/ci.yml

```diff
@@ -296,6 +296,19 @@ jobs:
           - name: dist-x86_64-linux
             <<: *job-linux-xl
 
+          - name: dist-aarch64-msvc
+            env:
+              RUST_CONFIGURE_ARGS: >-
+                --build=x86_64-pc-windows-msvc
+                --host=x86_64-pc-windows-msvc,aarch64-pc-windows-msvc
+                --target=x86_64-pc-windows-msvc,aarch64-pc-windows-msvc
+                --enable-full-tools
+                --enable-profiler
+              SCRIPT: python x.py dist
+              # RLS does not build for aarch64-pc-windows-msvc
```

Comment:
Out of curiosity: is there a tracking issue on RLS for why it doesn't build? I think I remember reading it's because of lack of support in winapi, no? rustc relies on winapi, so I'm curious what the issue is.

Reply:
I opened a tracking issue on RLS. https://github.com/rust-lang/rls/issues/1693